cmd/capibmadm/options/options.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package options contains the reusable and global variables.
+package options
+
+// IBMCloudAPIKeyEnvName holds the environmental variable name to set PowerVS service instance ID.
+const IBMCloudAPIKeyEnvName = "IBMCLOUD_API_KEY" //nolint:gosec
+
+var (
+	// IBMCloudAPIKey is IBM Cloud API key.
+	IBMCloudAPIKey string
+
+	// ServiceInstanceID is PowerVS service instance ID.
+	ServiceInstanceID string
+)
```

Comment:
what the better way of handling these flags - global params vs struct holding the options?!

Reply:
Moved to a global variable to a struct in options package.